<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>60</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;MS Reference Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>Area Hangar( mÂ²) :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cancelar">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>180</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
font: 90 20pt &quot;MS Shell Dlg 2&quot;;
  color:rgb(255, 255, 255);
  
	background-color: rgb(104, 127, 143);
border-radius:30px;
}
QPushButton:hover{
background-color: rgb(255, 0,0 );
	color:rgb(255, 255, 255);
}

QPushButton{

  color:rgb(255, 255, 255);
  
	
	background-color: rgb(141, 141, 141);
}
</string>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_area_hangar">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>100</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;MS Reference Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>ID del hangar:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_registrar">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>180</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
font: 90 20pt &quot;MS Shell Dlg 2&quot;;
  color:rgb(255, 255, 255);
  
	background-color: rgb(104, 127, 143);
border-radius:30px;
}
QPushButton:hover{
background-color: rgb(255, 0,0 );
	color:rgb(255, 255, 255);
}

QPushButton{

  color:rgb(255, 255, 255);
  
	
	background-color: rgb(141, 141, 141);
}
</string>
    </property>
    <property name="text">
     <string>Registrar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_id_hangar">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>* Campos obligatorios</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>70</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>621</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
